raster/src/main/scala/geotrellis/raster/io/geotiff/reader/TiffTagsReader.scala

```diff
@@ -105,7 +103,19 @@ object TiffTagsReader {
       case None =>
     }
 
-    tiffTags
+    // If it's undefined GDAL interprets the entire TIFF as a single strip
+    if(tiffTags.hasStripStorage) {
+        val rowsPerStrip =
+          (tiffTags
+            &|-> TiffTags._basicTags
+            ^|-> BasicTags._rowsPerStrip get).toInt
+        if (rowsPerStrip < 0) {
+          logger.warn("RowsPerStrip tag is not not defined, interpreting TIFF as a single strip.")
```

Comment:
`logger.debug` probably, its not wrong just unexpected semantics, right?

Reply:
Good question, i set up warn just because it is `warning` in GDAL.

Reply:
Fixed!

Reply:
GDAL is always right.